project/src/main/webapp/index.html

```diff
@@ -2,17 +2,26 @@
 <html>
   <head>
     <meta charset="UTF-8">
-    <title>My Portfolio</title>
-    <link rel="stylesheet" href="style.css">
-    <script src="script.js"></script>
+    <title>Demo</title>
+    <meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
+    <!-- Load TensorFlow.js. This is required to use the qna model. -->
+    <script src="https://cdn.jsdelivr.net/npm/@tensorflow/tfjs"></script>
+    <!-- Load the qna model. -->
+    <script src="https://cdn.jsdelivr.net/npm/@tensorflow-models/qna"></script>
+    <script type="text/javascript" src="api-key.js"></script>
+    <script type="text/javascript" src="client-id.js"></script>
+    <script src="https://maps.googleapis.com/maps/api/js?key=API-KEY&libraries=places"></script>
+    <script src = "script.js"></script>
+    <script src="classemailparse.js"></script>
+    <script src="search.js"></script>
+    <script src="maps.js"></script>
```

Comment:
This looks like the old html file. Can you pull origin master and then merge the new one before pushing it up?